CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## UNRELEASED
+
+### Changes
+
+* Webpack warnings about package size during the build process have been turned off by default.
```

Comment:
This should be updated to make clear they were only shut off for the admin UI build and they are still enabled for the public build, where a large javascript bundle can be problematic for SEO.

Reply:
Forgot to update when I changed the code in response to the first comment.